backend/tests/dbpm/1_divideupmarketpoolsharesdbpm_test.go

```diff
@@ -2,14 +2,15 @@ package test
 
 import (
 	"socialpredict/handlers/math/outcomes/dbpm"
+	"socialpredict/setup"
 	test "socialpredict/tests"
 	"testing"
 )
 
 func TestDivideUpMarketPoolSharesDBPM(t *testing.T) {
 	for _, tc := range test.TestCases {
 		t.Run(tc.Name, func(t *testing.T) {
-			yes, no := dbpm.DivideUpMarketPoolSharesDBPM(tc.Bets, tc.ProbabilityChanges)
+			yes, no := dbpm.DivideUpMarketPoolSharesDBPM(func() *setup.EconomicConfig { return setup.BuildInitialMarketAppConfig(t, .5, 1, 0, 0) }, tc.Bets, tc.ProbabilityChanges)
```

Comment:
this isn't passing, but does inject a basic default config for these tests, I need to figure out if they're dependent on a specific config type, then I can add it to the test cases. These tests should ultimately be migrated to the dbpm package though.